internals/generators/container/selectors.test.js.hbs

```diff
@@ -5,6 +5,6 @@
 
 describe('makeSelect{{ properCase name }}Domain', () => {
   it('Expect to have unit tests specified', () => {
-    expect('Test case').toEqual(false);
+    expect('true).toEqual(false);
```

Comment:
Found a redundant single quote in front of `true`, other than that, I support this PR.